common/ucf-faq-list-common.php

```diff
@@ -85,11 +114,11 @@ function ucf_faq_template( $template ) {
 	return $template;
 }
 
-add_filter( 'template_include', 'ucf_faq_template', 9 );
+add_filter( 'template_include', 'ucf_faq_topic_template', 9 );
 
 
 function ucf_faq_sort_order( $query ) {
-	if ( is_tax( 'topic' ) ) {
+	if ( is_tax( 'topic' ) || ( ( is_category() && get_query_var( 'post_type' ) === 'faq' )	|| is_post_type_archive() ) ) {
```

Comment:
The `is_category()` here shouldn't be necessary anymore, now that we're using the custom 'topic' taxonomy that is unique to FAQs.

The `is_post_type_archive()` check should also be passed the FAQ post type, so that we don't modify post type archives for other post types (`is_post_type_archive( 'faq' )`)

Could probably update this line to something like:
`if ( is_tax( 'topic' ) || ( get_query_var( 'post_type' ) === 'faq' && is_archive() ) || is_post_type_archive( 'faq' ) ) {`